README.md

```diff
@@ -1,3 +1,15 @@
  To use an emulated email server, Python provides one that is very handy that you can start in a
- second terminal with the following command: "$ python -m smtpd -n -c DebuggingServer localhost:8025"
- 
\ No newline at end of file
+ second terminal with the following command:
+ 
+ ```"$ python -m smtpd -n -c DebuggingServer localhost:8025"```
```

Comment:
Double quotes shuold be excluded here

Reply:
Yeah, and for one line comments you can use # also if you want